bin/recordium

```diff
@@ -28,9 +28,11 @@ parent_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 if os.path.basename(parent_dir) == 'recordium':
     # inside the project!!
     sys.path.insert(0, parent_dir)
+    os.chdir(parent_dir) #to allow use of relative paths
```

Comment:
When the software is runned from development, it can't find media icons because current directory is not correct. I fixed it